spec/kafka_spec.rb

```diff
@@ -9,6 +9,7 @@
         node.default['hadoop']['mapred_site']['mapreduce.framework.name'] = 'yarn'
         stub_command(/update-alternatives --display /).and_return(false)
         stub_command(/test -L /).and_return(false)
+        stub_command(/getent/).and_return(false)
```

Comment:
Default behavior didn't change, but tests were failing before I put these into the tests. I think ChefSpec needs any shell-based tests stubbed, whether it's in the execution or not. Dunno...